inference-engine/src/inference_engine/ie_core.cpp

```diff
@@ -9,23 +9,24 @@
 #include <mutex>
 #include <sys/stat.h>
 
+#include <auto_plugin/auto_config.hpp>
 #include <ie_core.hpp>
 #include <multi-device/multi_device_config.hpp>
-#include <ngraph/opsets/opset.hpp>
-#include <ngraph/ngraph.hpp>
 #include <ngraph/graph_util.hpp>
+#include <ngraph/ngraph.hpp>
```

Comment:
why so many includes changes, while the code changes are just strings manipulations?

Reply:
well, now we check not only Convolution but also its derivatives. like the above ilya's comment, the refer code is from: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/transformations/src/transformations/rt_info/primitives_priority_attribute.cpp#L30-L40

Reply:
from whta I see, we don't check the convolutions in the ie_core.cpp ( this file) only inside Auto

Reply:
sorry, I misunderstand, there should no change for headers, I think that is caused by automatically in Clion. I will modify and remove unnecessary change.